ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SettleDiscontinue-nonprod.json

```diff
@@ -40,6 +40,19 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "partiesNamesTD",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "APP-SOL-UNSPEC-PROFILE",
+          "APP-SOL-SPEC-PROFILE"
+        ],
+        "CRUD": "CRU"
```

Comment:
Shouldn't all the labels have read only permission?

Reply:
Updated to only Read permission. Agree.